CommonMC/src/EventWindowMarkerProducer_module.cc

```diff
@@ -26,42 +29,114 @@ namespace mu2e {
 
   class EventWindowMarkerProducer : public art::EDProducer {
     public:
-      explicit EventWindowMarkerProducer(fhicl::ParameterSet const& pset);
+      using Name=fhicl::Name;
+      using Comment=fhicl::Comment;
+      struct Config {
+        fhicl::Atom<float> systemClockSpeed{ Name("SystemClockSpeed"), Comment("Detector clock speed in MHz"), 40.0};
+        fhicl::Atom<float> constantOffset{ Name("ConstantTimeOffset"), Comment("Constant time offset correction in ns"), 0.0};
```

Comment:
Does this offset represent the shift of the DR marker WRT the proton bunch peak?  If so, does it make sense to name it as such?